Libraries/Opc.Ua.PubSub/Encoding/JsonDataSetMessage.cs

```diff
@@ -465,6 +465,7 @@ private void EncodeField(IJsonEncoder encoder, Field field)
                 valueToEncode = field.Value.StatusCode;
             }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
@mregen This statement hides the warning of using the obsolete UsingReversibleEncoding, shouldn't it be deleted and instead of UsingReversibleEncoding to use the UsingAlternateEncoding ? 
Is it intentionatly left there ?

Reply:
I didn't want to transform the Pubsub JSON sample to Compact/Verbose encoding yet, as spec isn't out yet.